cypress/integration/pre-notice_spec.js

```diff
@@ -18,8 +28,80 @@ describe('Pre notice form tests', function () {
         cy.contains('Tallenna ja lähetä');
         cy.contains('Tallenna luonnoksena');
 
+    });
+
+
+    it('save and send pre-notice', () => {
+        // Open pre notice form
+        cy.visit('/#/pre-notices');
+        cy.contains('Uusi muutosilmoitus').click();
```

Comment:
Here could be used an component id which was defined in previous task.

Reply:
Sorry, but I don't understand what do you mean.

Reply:
Sorry for confusion. :) Was commenting this when checking each commit separetely. 
In the first commit there was definition :id "add-new-pre-notice". Shouldn't it be used instead of 'Uusi muutosilmoitus'?

Reply:
It could be used. But we have used "label based" selectors more than id based so I think that we don't win anything by changing one of them.